rosdep/base.yaml

```diff
@@ -4583,6 +4584,7 @@ liborocos-bfl-dev:
 liborocos-kdl:
   debian:
     '*': [liborocos-kdl1.4]
+    bookworm: [liborocos-kdl1.5]
```

Comment:
The logic here should be inverted to list all the older distros at 1.4 and move the wildcard forward to 1.5 to cover bookworm such that if there's a new distro and liborocos-kdl hasn't changed version it will keep working.